### ISSUE TITLE
"OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect" ERROR MESSAGE

### ISSUE THREAD

**Issue by HoloDirk** [Anttwo/SuGaR#23]
On the final train.py command, I receive an error and for some reason "sugarfine_C:" is added to path BUT  does NOT exist as a file or folder in the directory - Final Output  -
....
Traceback (most recent call last):
  File "C:\Users\Dirk\SuGaR\train.py", line 160, in <module>
    refined_sugar_path = refined_training(refined_args)
  File "C:\Users\Dirk\SuGaR\sugar_trainers\refine.py", line 316, in refined_training
    os.makedirs(sugar_checkpoint_path, exist_ok=True)
  File "C:\Users\Dirk\anaconda3\envs\sugar\lib\os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "C:\Users\Dirk\anaconda3\envs\sugar\lib\os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "C:\Users\Dirk\anaconda3\envs\sugar\lib\os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
  [Previous line repeated 11 more times]
  File "C:\Users\Dirk\anaconda3\envs\sugar\lib\os.py", line 225, in makedirs
    mkdir(name, mode)
OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: 'C:\\Users\\Dirk\\SuGaR\\gaussian_splatting\\data\\head01\\sugarfine_C:'

**Comment by yuedajiong** [comment]
**_C,  it looks that is compiled CUDA.

1) compile and install gaussian_renderer.  (maybe you tested GS, you have compiled one.)
2) clean older gaussian_renderer. the path likes:   ~/.cache/torch_extensions/py310_cu117/gauss_rasterize/

**Comment by HoloDirk** [comment]
Thank you yuedajiong!

I actually re-installed from beginning. Got a new error

Found 3 issues that I have resolved:
1. pytorch3d did not work for me in sugar install, so I used facebook pytorch3d as install.
2. I put" data" folder in wrong directory, I placed in the gaussian_splatting directory NOT in SuGaR directory (correct position)
3. I forgot to create an "output" folder in the SuGaR directory

👍
